src/database/DataTypes/RequisitionItem.js

```diff
@@ -43,19 +44,21 @@ export class RequisitionItem extends Realm.Object {
     const { availableQuantity } = this.linkedTransactionItem;
     const { totalQuantity } = this.item;
 
-    return this.linkedTransactionItem ? availableQuantity : totalQuantity;
+    // TODO: this.linkedTransactionItem and this.item can both be falsey, unhandled possible return
+    if (availableQuantity) return availableQuantity;
```

Comment:
no need to refactor code in this PR, just applying eslint/prettier (can leave the comment though)

Reply:
Reverted.

Reply:
Reverted.